From 2ab56a59ca880eebb3b89704ca1952f388151d29 Mon Sep 17 00:00:00 2001
From: Chao Yu <yuchao0@huawei.com>
Date: Tue Nov 21 17:49:54 2017 +0800
Subject: [PATCH] f2fs: remove unneeded memory footprint accounting
Git-commit: 2ab56a59ca880eebb3b89704ca1952f388151d29
References: bsc#1106297
Patch-mainline: v4.16-rc1

We forgot to remov memory footprint accounting of per-cpu type
variables, fix it.

Fixes: 35782b233f37 ("f2fs: remove percpu_count due to performance regression")
Signed-off-by: Chao Yu <yuchao0@huawei.com>
Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>
Acked-by: Goldwyn Rodrigues <rgoldwyn@suse.com>

diff --git a/fs/f2fs/debug.c b/fs/f2fs/debug.c
index 4d92962..674f9bb 100644

--- a/fs/f2fs/debug.c
+++ b/fs/f2fs/debug.c
@@ -179,7 +179,6 @@ static void update_mem_info(struct f2fs_sb_info *sbi)
 	si->base_mem += sizeof(struct f2fs_sb_info) + sbi->sb->s_blocksize;
 	si->base_mem += 2 * sizeof(struct f2fs_inode_info);
 	si->base_mem += sizeof(*sbi->ckpt);
-	si->base_mem += sizeof(struct percpu_counter) * NR_COUNT_TYPE;
 
 	/* build sm */
 	si->base_mem += sizeof(struct f2fs_sm_info);
